wear/src/main/java/io/homeassistant/companion/android/views/ChooseEntityView.kt

```diff
@@ -150,3 +156,46 @@ private fun ChooseEntityChip(
         colors = getFilledTonalButtonColors()
     )
 }
+
+@Preview(device = WearDevices.LARGE_ROUND)
+@Composable
+fun ChooseEntityView_Empty_Preview() {
+    ChooseEntityView(
+        entitiesByDomainOrder = remember {
+            mutableStateListOf()
+        },
+        entitiesByDomain = remember {
+            mutableStateMapOf()
+        },
+        favoriteEntityIds = remember { mutableStateOf(listOf()) },
+        onNoneClicked = {},
+        onEntitySelected = {},
+        allowNone = true
+    )
+}
+
+@Preview(device = WearDevices.LARGE_ROUND)
+@Composable
+fun ChooseEntityView_WithData_Preview() {
+    ChooseEntityView(
+        entitiesByDomainOrder = remember {
+            mutableStateListOf("first", "second")
+        },
+        entitiesByDomain = remember {
+            mutableStateMapOf(
+                Pair(
+                    "first",
+                    mutableStateListOf(Entity("first", "on", "", Calendar.getInstance(), Calendar.getInstance(), null))
```

Comment:
Sorry just noticed you had some mock entities, we actually have some you can use for this purpose :)

https://github.com/home-assistant/android/blob/master/wear/src/main/java/io/homeassistant/companion/android/util/PreviewData.kt#L34

Reply:
Done